src/components/molecules/NavBar/components/SoundCloudRadio/SoundCloudRadio.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useMemo, useState } from "react";
+import classNames from "classnames";
+
+import { generateId } from "utils/string";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./SoundCloudRadio.scss";
+
+export type SoundCloudRadioProps = {
+  station: string | undefined;
+  volume: number;
+};
+
+export const SoundCloudRadio: React.FC<SoundCloudRadioProps> = ({
+  station,
+  volume,
+}) => {
+  const id = useMemo(() => generateId("SoundCloudRadio-player"), []);
+
+  const [visible, setVisible] = useState(false);
+
+  const toggleVisible = useCallback(
+    () => setVisible((prevState) => !prevState),
+    []
+  );
+
+  const parentClasses = classNames({
+    SoundCloudRadio: true,
+    "SoundCloudRadio--on": volume,
+    "SoundCloudRadio--off": !volume,
+    "SoundCloudRadio--visible": visible,
+    "SoundCloudRadio--invisible": !visible,
+  });
```

Comment:
```suggestion
  const parentClasses = classNames("SoundCloudRadio", {
    "SoundCloudRadio--on": volume,
    "SoundCloudRadio--off": !volume,
    "SoundCloudRadio--visible": visible,
    "SoundCloudRadio--invisible": !visible,
  });
```

Reply:
For this change, please first provide a reference to https://github.com/sparkletown/sparkle/blob/staging/docs/Questions.md (as per this precedent https://github.com/sparkletown/sparkle/pull/2219#discussion_r713837478 )